runtime/browser/android/intercepted_request_data.h

```diff
@@ -1,43 +0,0 @@
-// Copyright (c) 2012 The Chromium Authors. All rights reserved.
```

Comment:
How about keep these InterceptedRequestData related code?  The same as Xiao Song, we need to support public InterceptedRequestData shouldInterceptRequest(String url, boolean isMainFrame); to avoid fragmentation.